### DISCUSSION TITLE
Forgot Password Page

### DISCUSSION THREAD

**Discussion by 0786ShivaniBansal** [JanssenProject/jans#5960] [Q&A] [answered]
Hii Team as the default login.XHTML located in opt/jans/jetty/jans-auth/webapps contains 
```
`**<div class="form-group row">
<div class="col-sm-offset-3 offset-md-3 col-sm-7 col-md-7">
<div class="forgot_link">
<a href="/identity/person/passwordReminder.htm"
style="color: blue;"> <h:outputText
value="#{msgs['login.forgotYourPassword']}" />
</a>
</div>
</div>
</div>`**
```
![loginxhtml](https://github.com/JanssenProject/jans/assets/95275127/f2af6e94-10ef-4096-96d0-e3ad52ce6d88)
![redirection issue](https://github.com/JanssenProject/jans/assets/95275127/6a11a4ea-a14a-4c75-b637-7a77829ac280)

the link that is provided to redirect the passwordReminder.htm path is not located in jans there is no identity folder in janssen please look into it.
Thanks
Shivani Bansal

**Comment by 0786ShivaniBansal** [comment]
Hii Team is there any update regarding this waiting for the reply to proceed further.

Thanks

**Comment by nynymike** [comment]
This link should be removed. It's leftover cruft from Gluu 4. There is no `/identity` in Jans.

Opened issue in Jans: https://github.com/JanssenProject/jans/issues/5970

**Comment by 0786ShivaniBansal** [comment]
Hii @nynymike 
While providing the actual link where the forgot password.xhtml is located we are facing 403 forbidden error 
Even if we are replacing the file with another HTML file please provide us the troubleshooting steps.
Thanks

**Comment by nynymike** [comment]
There is no `/identity` endpoint in Jans. And there is no forgot password page... you'd have to build an Agama Flow for that and redirect to the `/authorize` endpoint with `acr_values=agama` and a specified `agama_flow` (for example, something like "forgot_password).

**Comment by nikita16code** [comment, marked as answer]
You write wrong file extension :-  " passwordReminder.htm "
that why it show an error.
it should be,
 " passwordReminder.html "
